stdlib/2and3/bz2.pyi

```diff
@@ -1,6 +1,34 @@
 # Stubs for bz2
 
-# TODO: This stub is incomplete
+from typing import Any, BinaryIO, TextIO, IO, Union
 
 def compress(data: bytes, compresslevel: int = ...) -> bytes: ...
 def decompress(data: bytes) -> bytes: ...
+
+def open(filename: Union[str, bytes, IO[Any]],
+         mode: str = 'rb',
+         encoding: str = None,
+         errors: str = None,
+         newline: str = None) -> Union[TextIO, BinaryIO]: ...
+
+class BZ2File(BinaryIO):
```

Comment:
Is this always going to be a binary file-like object? If it can be a text file, using `IO[Any]` as the base class is better.